### DISCUSSION TITLE
Focus Window and Center-Column on Mod+R/L-Mouse Click

### DISCUSSION THREAD

**Discussion by 456634** [YaLTeR/niri#535] [Ideas]
Hi,

First of all, thank you for this great wm. I came from Sway with many problems, and here everything runs smoothly, even with proprietary Nvidia drivers.

I work with a 34" monitor, and `center-column` is definitely something I have always missed. However, I think it would be cool if there was a way, for example, to click on a window while holding the mod key, and have it immediately centered. Currently, it's a bit cumbersome to first focus on the window or column and then call `center-column`. Would this be possible, or is it perhaps already possible and I just overlooked it?

Thank you very much for the great work.

**Comment by YaLTeR** [comment]
We need mouse click binds for this, and I guess for this to work, a potential mouse click bind should both focus the window and do the action?

**Comment by 456634** [comment]
Yes, that would be perfect. One could also use the binds to move or resize floating/sticky windows, for example, if this feature is implemented (I only followed the discussion peripherally).
